File: Python/1018.py
valor = int(input())
qntd = 0
print(valor)
while valor >= 100:
    qntd += 1
    valor -= 100
print(qntd, "nota(s) de R$ 100,00")
qntd = 0
while valor >= 50:
    qntd += 1
    valor -= 50
print(qntd, "nota(s) de R$ 50,00")
qntd = 0
while valor >= 20:
    qntd += 1
    valor -= 20
print(qntd, "nota(s) de R$ 20,00")
qntd = 0
while valor >= 10:
    qntd += 1
    valor -= 10
print(qntd, "nota(s) de R$ 10,00")
qntd = 0
while valor >= 5:
    qntd += 1
    valor -= 5
print(qntd, "nota(s) de R$ 5,00")
qntd = 0
while valor >= 2:
    qntd += 1
    valor -= 2
print(qntd, "nota(s) de R$ 2,00")
qntd = 0
while valor >= 1:
    qntd += 1
    valor -= 1
print(qntd, "nota(s) de R$ 1,00")

File: Javascript/1018.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let dinheiro = parseInt(lines.shift());
console.log(dinheiro);
console.log(Math.floor(dinheiro / 100), "nota(s) de R$ 100,00");
dinheiro = dinheiro % 100;
console.log(Math.floor(dinheiro / 50), "nota(s) de R$ 50,00");
dinheiro = dinheiro % 50;
console.log(Math.floor(dinheiro / 20), "nota(s) de R$ 20,00");
dinheiro = dinheiro % 20;
console.log(Math.floor(dinheiro / 10), "nota(s) de R$ 10,00");
dinheiro = dinheiro % 10;
console.log(Math.floor(dinheiro / 5), "nota(s) de R$ 5,00");
dinheiro = dinheiro % 5;
console.log(Math.floor(dinheiro / 2), "nota(s) de R$ 2,00");
dinheiro = dinheiro % 2;
console.log(dinheiro, "nota(s) de R$ 1,00");
